package.json

```diff
@@ -11,12 +11,12 @@
     "css:build": "cd ds-css && npm run build",
     "css:dist": "cp -rf ds-css/dist/css dist && cp README.MD dist && cp ds-css/package.json dist && replace-json-property ./dist/package.json version $npm_package_version",
     "website:init": "cd ds-ux-guidelines && npm install",
-    "website:build": "cd ds-ux-guidelines && npm run build:guide",
-    "website:prepare": "cp -rf ds-css/dist cbp-theme/kitchensink/ds-css && rsync -av --progress --exclude='node_modules' ds-ux-guidelines/ cbp-theme/kitchensink/design-system/",
+    "website:build": "cd ds-ux-guidelines && npm run build",
+    "website:prepare": "cp -rf ds-css/dist cbp-theme/kitchensink/ds-css && rsync -av --progress --exclude='node_modules' ds-ux-guidelines/build cbp-theme/kitchensink/design-system/",
```

Comment:
We need to update the website:prepare to ds-ux-guidelines/public/ since the new gatsby build uses the folder public instead

Reply:
Updated and pushed changes.